src/status_im2/contexts/onboarding/common/carousel/view.cljs

```diff
@@ -48,24 +50,56 @@
       {:static?            false
        :progress-bar-width progress-bar-width}]]))
 
+(def ^:const drag-threshold 100)
+(def ^:const progress-threshold 25)
+
+(defn get-last-progress
+  [progress]
+  (let [current-progress (reanimated/get-shared-value @progress)]
+    (if (>= current-progress drag-threshold)
+      drag-threshold
+      (* (+ (quot current-progress progress-threshold) 1) progress-threshold))))
+
+(defn get-next-progress
+  [progress]
+  (let [current-progress (reanimated/get-shared-value @progress)]
+    (if (< current-progress progress-threshold)
+      0
+      (* (- (quot current-progress progress-threshold) 1) progress-threshold))))
+
+(defn drag-gesture
+  [progress paused]
+  (->
+    (gesture/gesture-pan)
+    (gesture/enabled true)
+    (gesture/max-pointers 1)
+    (gesture/on-finalize
+     (fn [event]
+       (when (> (oops/oget event "translationX") drag-threshold)
+         (animation/update-progress progress paused (get-next-progress progress)))
+       (when (< (oops/oget event "translationX") (- drag-threshold))
+         (animation/update-progress progress paused (get-last-progress progress)))))))
+
 (defn f-view
-  [{:keys [animate? progress header-text background header-background]}]
+  [{:keys [animate? progress paused header-text background header-background]}]
   (let [window-width       (rf/sub [:dimensions/window-width])
         status-bar-height  (:status-bar-height @navigation/constants)
         progress-bar-width (- window-width 40)
         carousel-left      (animation/carousel-left-position window-width animate? progress)
         container-view     (if animate? reanimated/view rn/view)]
     [:<>
-     [container-view {:style (style/carousel-container carousel-left animate?)}
-      (for [index (range 2)]
-        ^{:key index}
-        [content-view
-         {:window-width      window-width
-          :status-bar-height status-bar-height
-          :index             index
-          :header-text       header-text
-          :header-background header-background}
-         (when background background)])]
+     [gesture/gesture-detector
+      {:gesture (drag-gesture progress paused)}
```

Comment:
a simple way to make this non-swipeable is to do something like adding a prop 
`swipeable?` and then here have:
`{:gesture (when swipeable? (drag-gesture progress paused))}`

checked out your branch and this worked for me. This should avoid issues on the identifiers page then 👍